core/src/banking_stage/leader_slot_metrics.rs

```diff
@@ -19,33 +20,22 @@ use {
 /// counted in `Self::retryable_transaction_indexes`.
 /// 2) Did not execute due to some fatal error like too old, or duplicate signature. These
 /// will be dropped from the transactions queue and not counted in `Self::retryable_transaction_indexes`
-/// 3) Were executed and committed, captured by `committed_transactions_count` below.
-/// 4) Were executed and failed commit, captured by `failed_commit_count` below.
+/// 3) Were executed and committed, captured by `transaction_counts` below.
+/// 4) Were executed and failed commit, captured by `transaction_counts` below.
 pub(crate) struct ProcessTransactionsSummary {
     // Returns true if we hit the end of the block/max PoH height for the block before
     // processing all the transactions in the batch.
     pub reached_max_poh_height: bool,
 
-    // Total number of transactions that were passed as candidates for execution. See description
-    // of struct above for possible outcomes for these transactions
-    pub transactions_attempted_execution_count: usize,
-
-    // Total number of transactions that made it into the block
-    pub committed_transactions_count: usize,
-
-    // Total number of transactions that made it into the block where the transactions
-    // output from execution was success/no error.
-    pub committed_transactions_with_successful_result_count: usize,
-
-    // All transactions that were executed but then failed record because the
-    // slot ended
-    pub failed_commit_count: usize,
+    // Total transaction counts tracked for reporting `LeaderSlotMetrics`. See
```

Comment:
consistent with other comments so fine to leave as is, but why would all the comments on theese fields not be doc-comments?